Formula/webpod.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class Webpod < Formula
+  desc "Deploy websites and apps anywhere"
+  homepage "https://github.com/webpod/webpod"
+  url "https://registry.npmjs.org/webpod/-/webpod-1.0.0.tgz"
```

Comment:
one nitpick comment, your release tags (https://github.com/webpod/webpod/releases) do not match with npmjs artifact

Reply:
True! Thanks for noticing. Will fix it.